examples/full-example/.mendelrc

```diff
@@ -210,4 +209,6 @@ env:
         outfile: server.js
         entries:
           - ./main.js
-        runtime: main # This
+        # If npm package has multiple entry files, this pulls only
+        # server side files defined in "main" field of package.json
+        runtime: main
```

Comment:
I think this is a remaining design flaw from my initial mendel-v2-rc draft. Think about it: All transforms are per instance config, all types are per "id" config. All generators and outlets should be the same. You should reference and outlet or generator per ID of it's instance and not have a generator perform differently based on the bundle config.

This is a good candidate for a small revision on the `.mendelrc` spec. For now, we could follow example of the other docs, declaring a bundle and what properties are bundle specific and mendel specific, than we could say: "other properties are used by either a generator or an outlet, please refer to ....".